src/App.vue

```diff
@@ -17,13 +22,68 @@ export default {
 }
 </script>
 
+
 <style>
+
+@font-face {
+  font-family: "Coiny";
+  src: url('https://fonts.googleapis.com/css?family=Coiny');
+}
+
+body{
+  
+  margin: 0;
+  padding: 0;
+}
+
 #app {
-  font-family: 'Avenir', Helvetica, Arial, sans-serif;
+  font-family: 'Coiny', cursive;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   text-align: center;
   color: #2c3e50;
-  margin-top: 60px;
 }
+
+input, button, .form-submit-button{
+  font-family: 'Coiny', cursive;
+}
+
+.slogan{
+  font-style:italic;
```

Comment:
missing space : `font-style: italic;`
You should probably add a linter. See [prettier](https://github.com/prettier/prettier)